content/departments/product-engineering/engineering/process/releases/release_issue_template.md

```diff
@@ -102,7 +111,7 @@ Once there are no more release-blocking issues (as reported by the `release:stat
   yarn release release:close
   ```
 
-## Post-release
+### Post-release
 
 Notify the next [release captain](./index.md#release-captain) that they are on duty for the next release. They should complete the steps in this section.
```

Comment:
> They should complete the steps in this section.

Is this really true? It feels a bit weird that this issue can't be closed until the next release captain does some work.

Reply:
Updated!